### ISSUE TITLE
additional fields on sign up page

### ISSUE THREAD

**Issue by holaymolay** [codeclubsocial/codeclub_website_dev#23]
SignUp page needs a couple of more field about the user information, like name.

**Comment by holaymolay** [comment]
Craig was working on signup email validation and this issue came up.. 

The sign up page now contains the following:
- Username
- First Name
- Last Name
- Email
- Password
- Confirm Password

**Comment by holaymolay** [comment]
- Username needs better validation, prohibiting spaces (`&nbsp;`)
- First and Last names need to be added to database
- Password fields need validation. This will be handled via js within the DOM.

**Comment by craig429** [comment]
Okay, username cannot contain spaces at sign up.
First/Last name fields have been added. At this point they can be blank.
Login validation happens on username/password instead of email/password. That actually is not part of this issue but I figured that's where we're going.
I'll leave the password validation to Brian, unless you want me to take that on (gotta find out what a DOM is first). Once that's done we can close this.
Today's latest commit to `craig-email-verify` contains all this cool stuff.
